pkgs/servers/monitoring/cockpit/default.nix

```diff
@@ -215,7 +221,10 @@ stdenv.mkDerivation rec {
     npm run stylelint
   '';
 
-  passthru.tests = { inherit (nixosTests) cockpit; };
+  passthru = {
+    tests = { inherit (nixosTests) cockpit; };
+    updateScript = ./update.sh;
```

Comment:
```suggestion
```
the bot will use nix-update which does the same thing

Reply:
I didn't know about that

BTW, this bot is a bit obscure IMO, so I wanted to be a bit more explicit about the update behavior.